Env/env.go

```diff
@@ -0,0 +1,48 @@
+package env
```

Comment:
Recommend changing the package name to something more non-generic (`env` is a common name, with meaning at systems level).

Maybe call it: `l2env`